lib/OpenQA/Utils.pm

```diff
@@ -1211,6 +1211,25 @@ sub service_port {
     return $base + $offsets->{$service};
 }
 
+sub rndstr {
```

Comment:
Since you're already cleaning up things you could give this function (and the next one) a better name, e.g. `generate_random_string`.